atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -564,6 +564,7 @@ def _set_worker_arg(self, replacement_pullspecs):
             for repl in replacement_pullspecs
             if repl['replaced']
         }
+        # OSBS2 TBD
```

Comment:
Why is TBD here?

Reply:
Because in `override_build_kwarg(self.workflow, "operator_bundle_replacement_pullspecs", arg)`, that `override_build_kwarg` is imported from 'build_orchestrate_build', which is supposed to be getting deleted.